test/System.Net.Http.Formatting.Test/Handlers/ProgressMessageHandlerTest.cs

```diff
@@ -84,7 +84,15 @@ public async Task SendAsync_InsertsReceiveProgressWhenResponseEntityPresent(bool
                 }
                 else
                 {
+#if NET6_0_OR_GREATER
+                    // Test initialization (request.CreateResponse() near the end of this file) for this case leaves
+                    // response.Content non-null (an EmptyContent or StreamContent instance). Those types are internal.
+                    // Just confirm the length matches expectations.
+                    Assert.NotNull(response.Content);
+                    Assert.Equal(0L, response.Content.Headers.ContentLength);
```

Comment:
@stephentoub and @Tratcher this is one of two test areas I mentioned in our email thread&hellip;